charts/qiskit-serverless/charts/gateway/templates/rayclustertemplate.yaml

```diff
@@ -119,6 +119,9 @@ data:
               - mountPath: /data
                 name: user-storage
                 subPath: {{`{{ user_id }}`}}
+              - mountPath: /function_data
+                name: user-storage
```

Comment:
It may be worth an internal discussion on whether we want / need to use a different volume for storing provider data

Reply:
Yeah, I agree I'm not sure about this solution either.

Reply:
Yeah, I realized that it may not necessary when I implemented this.